lib/core/src/restore.rs

```diff
@@ -370,8 +437,14 @@ impl LiquidSdk {
                         false => (Some(second_tx_id), Some(first_tx_id)),
                     }
                 }
+                0 => {
+                    debug!(
+                        "No BTC script history found while recovering data for Chain Send Swap {swap_id}"
+                    );
+                    (None, None)
+                }
                 n => {
-                    error!("BTC script history with unexpected length {n} found while recovering data for Chain Send Swap {swap_id}");
+                    info!("BTC script history with unexpected length {n} found while recovering data for Chain Send Swap {swap_id}");
```

Comment:
```suggestion
                    warn!("BTC script history with unexpected length {n} found while recovering data for Chain Send Swap {swap_id}");
```

Same comment as above about log severity, as this is an edge-case. Maybe `warn` is more appropriate?